.github/semantic.yml

```diff
@@ -185,7 +185,6 @@ scopes:
 
   # sinks
   - amqp sink # Anything `amqp` sink related
-  - apex sink # Anything `apex` sink related
```

Comment:
Needs to be removed in the TF resources

Reply:
:thinkes: TF resources?

Reply:
https://github.com/vectordotdev/resources/blob/master/vector/github/semantic_pull_requests.tf

Reply:
That file indicates it is autogenerated ...

Reply:
The file it creates says it's autogenerated and to go to that link to update it

Reply:
https://github.com/vectordotdev/resources/pull/192